ui/src/model-manager/missing-models-drawer/MissingModelItem.tsx

```diff
@@ -0,0 +1,91 @@
+import { GridItem, VStack, Button, Text } from "@chakra-ui/react";
+import { IconExternalLink } from "@tabler/icons-react";
+import { useEffect, useState } from "react";
+
+export interface MissingModel {
+  class_type: string;
+  input_name: string;
+  received_value: string;
+}
+
+interface Props {
+  model: MissingModel;
+}
+
+export default function MissingModelItem({ model }: Props) {
+  const [suggestedUrls, setSuggestedUrls] = useState<{ url: string, name: string }[]>([]);
+  const modelName = formatSearchQuery(model.received_value);
+  const getSearchData = async () => {
+    // huggingface search
+    const hfData = await fetch(`https://huggingface.co/api/models?limit=3&search=${modelName}`);
+    const hfSearchResult = await hfData.json() as { id: string; modelId: string }[];
+    const hfUrls = hfSearchResult.map(({ modelId }) => ({ name: `${modelId.split('/')[1]} on HuggingFace`, url: `https://huggingface.co/${modelId}` }));
+    console.log(modelName, "hf", hfSearchResult, hfUrls);
+    // civitai search
+    const civitaiData = await fetch(`https://civitai.com/api/v1/models?limit=3&query=${modelName}`);
+    const civitaiSearchResult = await civitaiData.json() as { items: { id: string, name: string }[] };
+    const civitaiUrls = civitaiSearchResult.items?.map(({ name, id }) => ({ name: `${name} on civitAI`, url: `https://civitai.com/models/${id}` })) ?? [];

```

Comment:
i think you should parallel these 2 search request (hf and civitiai) once hf is done directly render the hf button, 

 the civitiai call is slow and is blocking the hf button rendering I feel